lib/Doctrine/Common/DataFixtures/Executor/AbstractExecutor.php

```diff
@@ -141,12 +141,7 @@ public function purge()
         $this->purger->purge();
     }
 
-    /**
-     * Executes the given array of data fixtures.
-     *
-     * @param array $fixtures Array of fixtures to execute.
-     * @param bool  $append   Whether to append the data fixtures or purge the database before loading.
-     */
+    /** @inheritDoc */
     abstract public function execute(array $fixtures, $append = false);
```

Comment:
*If* we keep the `Executor` interface, you can remove the `execute()` method here.